morgoth/utils/result_reader.py

```diff
@@ -295,6 +308,108 @@ def _read_post_equal_weights_file(self, post_equal_weights_file):
+        df = df.astype({"Peak": int})
+        df_filtered = df[df['Peak']>400]
+        
+        #for table of catalog
+        table = Table.from_pandas(df_filtered)
+        #table.show_in_browser(jsviewer=True)
+        
+        #transform input in SkyCoord
+        position = SkyCoord(self._ra*unit.deg, self._dec*unit.deg, frame="icrs")
+        
+        #transform table data in SkyCoord
+        coords = []
+        for i in range(len(df_filtered['RA J2000 Degs'])):
+            ra = table[i]['RA J2000 Degs']  
+            dec = table[i]['Dec J2000 Degs']
+            coords.append(SkyCoord(ra*unit.deg, dec*unit.deg, frame="icrs"))
+            
+        #get separation value
+        separations = []
+        for i in coords:
+            z = i.separation(position)
+            separations.append(z.to(unit.deg))
+            
+        #for table of separations
+        table["Separation Degs"] = separations
+        table.round(3)
+        table.sort("Separation Degs")
+        #table.show_in_browser(jsviewer=True)
+            
+        #create dictionary
+        dic = {}
+        for i in range(3):
+            dic[table[i]['Source Name']]={"ra":table[i]['RA J2000 Degs'],
+            "dec":table[i]['Dec J2000 Degs'], "separation":table[i]["Separation Degs"]}
+
+        self._dic_bright_sources = dic
+
+
+    def _sep_SGRs(self):
+
+        #get csv data from website
+        url = 'http://www.physics.mcgill.ca/~pulsar/magnetar/TabO1.csv'
+        r = requests.get(url, allow_redirects=True)
+        open('SGRList.csv', 'wb').write(r.content)
+        df = pd.read_csv('SGRList.csv')
+
+        #for table of catalog
+        table = Table.from_pandas(df)
+    
+        #transform table data in SkyCoord
+        coords = []
+        for i in range(len(df['RA'])):
+            hour_ra = Angle(table[i]['RA']+" hours")
+            ra = hour_ra.to(unit.deg)
+            arc_dec = table[i]['Decl']
+            dec = Angle(tuple(map(float,arc_dec.split(' '))), unit=unit.deg)
+            coords.append(SkyCoord(ra, dec, frame="icrs"))
+                
+        #transform input in SkyCoord
+        position=SkyCoord(self._ra*unit.deg, self._dec*unit.deg, frame="icrs")
+        
+        #get separation value
+        separations = []
+        for i in coords:
+            z = i.separation(position)
+            separations.append(z.to(unit.deg))
+        
+        #for table of separations
+        table["Separation Degs"] = separations
+        table["Coords Degs"] = coords
+        table.round(3)
+        table.sort("Separation Degs")
+        #table.show_in_browser(jsviewer=True)
+        
+        #create dictionary
+        dic = {}
+        for i in range(3):
+            dic[table[i]['Name']]={"ra":round(table[i]['Coords Degs'].ra.degree,3),
+            "dec":round(table[i]['Coords Degs'].dec.degree,3),
+            "separation":table[i]["Separation Degs"]}
+
+        self._dic_SGRs = dic
```

Comment:
change this to

```
dic = {}
for i in range(3):
    dic[str(table[i]['Name'])]={"ra":float(round(table[i]['Coords Degs'].ra.degree,3)),
                                "dec":float(round(table[i]['Coords Degs'].dec.degree,3)),
                                "separation":float(table[i]["Separation Degs"])}
self._dic_SGRs = dic
```

Same for bright sources. Otherwise the numbers were saved as some strange type in the yaml and could not be read later. I changed this when we added the table on the website, but let's add it here to make it consistent.